### PR TITLE
GRW-1954/Empty cart message secondary text

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

- Small change to make the 'your cart is empty' text a bit lighter to match the design. aka using the `textSecondary` token.

Screenshot 2022-12-20 at 14 09 09

## Checklist before requesting a review

- [x] I have performed a self-review of my code

[GRW-1954]: https://hedvig.atlassian.net/browse/GRW-1954?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/store/src/components/CartPage/CartPage.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const EmptyState = ({ children, prevURL }: EmptyStateProps) => {
         <Space y={2}>
           <Space y={1}>
             <CenteredText>¯\_(ツ)_/¯</CenteredText>
-            <CenteredText>{t('CART_EMPTY_SUMMARY')}</CenteredText>
+            <CenteredTextSecondary>{t('CART_EMPTY_SUMMARY')}</CenteredTextSecondary>
```
Comment (robinandeer) [inline apps/store/src/components/CartPage/CartPage.tsx:79]: I think we can just use props for this nowadays :D

```suggestion
            <Text align="center" color="textSecondary">{t('CART_EMPTY_SUMMARY')}</Text>
```

**Review by robinandeer — APPROVED** [review]
Cool! I think we can slightly simplify even

File: apps/store/src/components/CartPage/CartPage.tsx (hunk 1 above)
Comment (gustaveen) [inline apps/store/src/components/CartPage/CartPage.tsx:79]: We can use props for `color` and `alignment` int this case  😃
```suggestion
            <Text color="textSecondary" align="center">{t('CART_EMPTY_SUMMARY')}</Text>
```

Comment (gustaveen) [reply apps/store/src/components/CartPage/CartPage.tsx:79]: haha sorry, didn't see @robinandeer comment 😁

Comment (KarnellSchultz) [reply apps/store/src/components/CartPage/CartPage.tsx]: Tack! did not know it was a thing 🧠

**Pull request merged**